c/parallel/ztest_parallel_for.cc

```diff
@@ -102,7 +102,7 @@ void test_parallel_for_ordered(size_t n) {
 
   dt::parallel_for_ordered(
     /* n_iters = */ n,
-    /* n_threads = */ size_t(-1),
+    /* n_threads = */ dt::NThreads(size_t(-1)),
```

Comment:
this is same as `dt::NThreads()`

Reply:
Yeah, didn't notice this was just a way to get all the available threads. Expected to see something like `dt::num_threads_in_pool()` here. Anyways, simplified this to `dt::NThreads()`.